src/components/addAssetsDialog/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { Button, VStack } from '@chakra-ui/react';
+
+import { useUpdateSettingsRequest } from '@/modules/settings/hooks';
+import { useWorkspaceContext } from '@/modules/workspace/WorkspaceProvider';
+
+import { Dialog } from '../dialog';
+import { BridgeIcon, CoinsIcon, DownLeftArrow } from '../icons';
+import WelcomeCard from '../welcomeDialog/card';
+
+interface IWelcomeDialogProps {
+  isOpen: boolean;
+  setIsAddAssetDialogOpen: (value: boolean) => void;
+  setIsDepositDialogOpen: (value: boolean) => void;
+}
+
+const AddAssetsDialog = ({
+  isOpen,
+  setIsDepositDialogOpen,
+  setIsAddAssetDialogOpen,
+}: IWelcomeDialogProps) => {
+  const {
+    screenSizes: {
+      isMobile,
+      isSmall,
+      isLitteSmall,
+      isLowerThanFourHundredAndThirty,
+    },
+    authDetails: {
+      userInfos: { id, refetch },
+    },
+  } = useWorkspaceContext();
+
+  const updateUserMutation = useUpdateSettingsRequest();
+
+  const handleUpdateUser = async () => {
+    updateUserMutation.mutate(
+      { id, first_login: false },
+      { onSuccess: () => refetch() },
+    );
+  };
+
+  const handleOpenDepositDialog = () => {
+    handleUpdateUser();
+    setIsDepositDialogOpen(true);
+    setIsAddAssetDialogOpen(false);
+  };
+
+  const handleClose = () => {
+    handleUpdateUser();
+    setIsAddAssetDialogOpen(false);
+  };
+
+  const handleRedirectToMainNet = async () => {
+    window.open('https://app-mainnet.fuel.network/bridge', '_BLANK');
```

Comment:
add and environment varible to this link `'https://app-mainnet.fuel.network/bridge'`